### DISCUSSION TITLE
Nice console clear method

### DISCUSSION THREAD

**Discussion by iiPythonx** [Textualize/rich#1177] [Q&A]
I've been exploring a bit with rich, and it works well although I would like to know something.

For one, is there a way for rich to update the screen without reprinting, as in editing an already printed message.
Second, if there isn't, I've been using the windows `cls` command but doing that too fast makes the terminal do a little blink-looking animation which is extremely hard on the eyes.

Is there a nicer way to clear the console without giving that flash?

**Comment by willmcgugan** [comment]
You can clear the console with `Console.clear` but that isn't the best way of creating an animation. See [Live Display](https://rich.readthedocs.io/en/latest/live.html) for a better way.

**Reply by iiPythonx** [reply]
Sure, I'll give that a look.
